embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/internal/clock/ClockTest.kt

```diff
@@ -0,0 +1,22 @@
+package io.embrace.android.embracesdk.internal.clock
+
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotEquals
+import org.junit.Test
+
+internal class ClockTest {
+    @Test
+    fun `test timestamp fallback conversion`() {
+        // We need to fix this before the year 5000. I will be dead by then so it'll be someone else's problem.
```

Comment:
Can a user set device year to >5000 & therefore trigger this use-case? What is the consequence of doing this?

Reply:
Actually, I should detect the time based on the SDK's epoch time to determine how far in the future to look. Let me do this in another PR - but that's a good shout!